splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1343,7 +1355,11 @@ public boolean isMetadataAccessRestrictionEnabled() {
 
     @Override
     public void setMetadataAccessRestrictionEnabled() {
-        metadataAccessRestrictionEnabled = SIDriver.driver().getConfiguration().getMetadataRestrictionEnabled();
+        String metadataRestriction =
+                SIDriver.driver().getConfiguration().getMetadataRestrictionEnabled();
+        metadataAccessRestrictionEnabled =
+                metadataRestriction.equals(SQLConfiguration.METADATA_RESTRICTION_NATIVE) ||
+                        metadataRestriction.equals(SQLConfiguration.METADATA_RESTRICTION_RANGER);
         SpliceLogUtils.trace(LOG,"metadataAccessRestritionEnabled=%b",metadataAccessRestrictionEnabled);
```

Comment:
Could you help to change SpliceLogUtils.trace() to SpliceLogUtils.info(), this way, we can see the value during the boot time with the default logging option. 
Could you also print the value metadataRestriction directly? Thanks!

Reply:
@yxia92  Fixed this